src/python/gudhi/tensorflow/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .cubical_layer import CubicalLayer
+from .lower_star_simplex_tree_layer import LowerStarSimplexTreeLayer
+from .rips_layer import RipsLayer
```

Comment:
So it is impossible to import say just CubicalLayer without also importing all the other tensorflow layers?
That's a possible choice, I am just wondering if that's deliberate.

Reply:
That is not really deliberate, I just don't know how to avoid this -_-'

Reply:
Uh? You mean what happens if you remove those 3 lines? :confused:
Or are you asking how to have CubicalLayer directly in gudhi.tensorflow but still require some `import gudhi.tensorflow.cubical_layer` or something?
(again, I am not asking you to change that, just discussing what we want in the end)

Reply:
Actually I wanted to be able to do `from gudhi.tensorflow import *`, but I don't know if it is possible to do `from gudhi.tensorflow import CubicalLayer` without also importing the other two. That being said, I do not really care about this `__init__` file, if there is a better, more consistent way of writing it, I'll be happy to use it.

Reply:
I think what you describe forces to load everything. To be able to load only some, I think it would need to be `from gudhi.tensorflow.cubical_layer import CubicalLayer` (there could still be a short `from gudhi.tensorflow.all import *` for those who want everything). I don't know if doing that would be useful, or if we should stick with loading everything, which is simpler, if a bit wasteful.